src/DynamoCoreWpf/ViewModels/Search/SearchMemberGroup.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Dynamo.Core;
+using Dynamo.UI;
+using Dynamo.Wpf.ViewModels;
+
+namespace Dynamo.Search
+{
+    public class SearchMemberGroup : NotificationObject
+    {
+        private List<NodeSearchElementViewModel> members;
+
+        public string FullyQualifiedName { get; private set; }
+
+        public string Prefix
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(FullyQualifiedName))
+                    return string.Empty;
+
+                int index = FullyQualifiedName.IndexOf(delimiter);
+                var name = index > -1 ? FullyQualifiedName.Substring(index + delimiter.Length)
+                                      : FullyQualifiedName;
+
+                index = name.LastIndexOf(delimiter);
+                return index > -1 ? name.Substring(0, index + delimiter.Length)
+                                  : string.Empty;
+            }
+        }
+
+        public string GroupName
+        {
+            get
+            {
+                // Skip past the last delimiter and get the group name.
+                int index = FullyQualifiedName.LastIndexOf(delimiter);
+                return index > -1 ? FullyQualifiedName.Substring(index + delimiter.Length)
+                                  : string.Empty;
+            }
+        }
+
+        public IEnumerable<NodeSearchElementViewModel> Members
+        {
+            get
+            {
+                if (!showAllMembers)
+                    return members;
+
+                if (members.Count == 0) return new List<NodeSearchElementViewModel>();
```

Comment:
Just noticed this, why don't we simply return `members` here, since it is the right type and it is empty? Then all the commented out codes can be removed, too.